framework/egl/wrapper/eglwEnums.inl

```diff
@@ -238,6 +238,7 @@
 #define EGL_LOSE_CONTEXT_ON_RESET_EXT						0x31BF
 #define EGL_PLATFORM_X11_EXT								0x31D5
 #define EGL_PLATFORM_X11_SCREEN_EXT							0x31D6
+#define EGL_PLATFORM_WAYLAND_KHR							0x31D8
```

Comment:
You should include all modifications that re-running scripts/gen_egl.py makes as part of the commit.